### PR TITLE
docs(changelog): Provide off-the-shelf dashboard

### PR DESCRIPTION
**Pull request by tomkerkhove**
Provide off-the-shelf dashboard.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)

Relates to #3911

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### New
 
+- **General**: Provide off-the-shelf Grafana dashboard for application autoscaling ([#3911](https://github.com/kedacore/keda/issues/3911))
```
Comment (JorTurFer) [inline CHANGELOG.md:41]: Place this alphabetically sorted 🙏

Comment (tomkerkhove) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -50,6 +50,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 - **Azure Service Bus Scaler**: Add support for Shared Access Signature (SAS) tokens for authentication. ([#2920](https://github.com/kedacore/keda/issues/2920))
 - **Azure Service Bus Scaler:** Support regex usage in queueName / subscriptionName parameters. ([#1624](https://github.com/kedacore/keda/issues/1624))
 - **ElasticSearch Scaler**: Support for ElasticSearch Service on Elastic Cloud ([#3785]https://github.com/kedacore/keda/issues/3785)
+- **Operations**: Provide off-the-shelf Grafana dashboard for application autoscaling ([#3911](https://github.com/kedacore/keda/issues/3911))
```
Comment (zroubalik) [inline CHANGELOG.md:53]: `Operations`? If it is another general changelong entry type, then I'd move it to the top below General section, or actually I'd mark this as `General`.

Comment (tomkerkhove) [reply CHANGELOG.md]: Done and added link to where docs will be

**Review by zroubalik — APPROVED** [review]
nice

**Pull request merged**